_posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md

```diff
@@ -4,7 +4,7 @@ comments: true
 date: 2017-08-11 01:01:54+00:00
 layout: post
 link: http://www.96boards.org/blog/self-balancing-bot-using-96boards-part1/
-slug: introducing-self-balancing-bot-using-96boards
+slug: self-balancing-bot-using-96boards-part1
 featured_image: mpu6050.jpg
 thumbnail_image: mpu6050-thumb.jpg
```

Comment:
Instruction on the help document
https://github.com/96boards/website/blob/master/help/blog.md#custom-thumbnails